### PR TITLE
Upstreams  September

### PR DESCRIPTION
**Pull request by doob187**
Add all here for upstream pushes for September upstream:

### CODE REVIEW

**Comment by aelfa** [comment]
@DrAg0n141 Organizr already added under:
apps/selfhosted/organizr.yml

**Comment by drag0n141** [comment]
> @DrAg0n141 Organizr already added under:
> apps/selfhosted/organizr.yml

Thanks delete :)

File: apps/addons/dashy.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+    image: 'lissy93/dashy:latest'
+    restart: '${RESTARTAPP}'
+    networks:
+      - proxy
+    security_opt:
+      - 'no-new-privileges:true'
+    volumes:
+      - '${APPFOLDER}/dashy/conf.yml:/app/public/conf.yml:rw'
```
Comment (aelfa) [inline apps/addons/dashy.yml:20]: @DrAg0n141 Please fix this!

**Review by aelfa — COMMENTED** [review]
Needs Sub-action.

File: apps/addons/dashy.yml (hunk 1 above)
Comment (doob187) [reply apps/addons/dashy.yml:20]: Thiw will be super hard to code 
It's so massively customizable parts 

So I thinks is mus show the link  and  the after press enter just create a empty file and open them with  

```sh
sudo nano ${APPFOLDER}/dashy/conf.yml
```

All other should be made to much problems and you lose the custome parts

**Comment by doob187** [comment]
Added dockserver docker updater

**Review by doob187 — COMMENTED** [review]
Add fail-safe to change the owner permission

**Comment by aelfa** [comment]
> Included static IPs for multiple sections with the following details.


1. **mediamanager**:         `172.18.0.100-123`
2. **downloadclients**:      `172.18.0.124-137`
3. **coding**:               `172.18.0.138-139`
4. **system**:               `172.18.0.140-146`
5. **traefik+authelia etc**: `172.18.0.250-254`
6. **mediaserver**:          `172.18.0.220-223`

File: apps/system/watchtower.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+services:
+  watchtower:
+    hostname: 'watchtower'
+    container_name: 'watchtower'
+    image: 'ghcr.io/containrrr/watchtower:latest'
+    environment:
+      - 'PUID=${ID}
+      - 'PGID=${ID}'
+      - 'TZ=${TZ}'
+      - 'WATCHTOWER_CLEANUP=true'
```
Comment (doob187) [inline apps/system/watchtower.yml]: Environment  !!!!

**Review by doob187 — COMMENTED** [review]
See notes

File: apps/system/watchtower.yml (hunk 2 above)
Comment (drag0n141) [reply apps/system/watchtower.yml]: For Cleanup?

Comment (doob187) [reply apps/system/watchtower.yml]: It's better for the hole part 

The goal is ..  all can be changed  <| the goal of DS |>

**Review by doob187 — COMMENTED** [review]
Env?!
```sh

- 'WATCHTOWER_REMOVE_VOLUMES=true' 
- 'WATCHTOWER_INCLUDE_STOPPED=false' 
- 'WATCHTOWER_NO_STARTUP_MESSAGE=true'@@ -18,7 +18,7 @@ services:

 - 'WATCHTOWER_ROLLING_RESTART=false'
 -  - 'WATCHTOWER_TIMEOUT=180s'

```

**Comment by aelfa** [comment]
> 
> 
> Env?!
> 
> ```shell
> - 'WATCHTOWER_REMOVE_VOLUMES=true' 
> - 'WATCHTOWER_INCLUDE_STOPPED=false' 
> - 'WATCHTOWER_NO_STARTUP_MESSAGE=true'@@ -18,7 +18,7 @@ services:
> 
>  - 'WATCHTOWER_ROLLING_RESTART=false'
>  -  - 'WATCHTOWER_TIMEOUT=180s'
> ```

@doob187 [`acf58db` (#203)](https://github.com/dockserver/dockserver/pull/203/commits/acf58dbe3cde77af9e5aa7e85d6db16afb71af22)

**Comment by aelfa** [comment]
@doob187 @DrAg0n141 The basic-proxy network is defined but not created.

**Comment by doob187** [comment]
https://github.com/dockserver/dockserver/blob/241f2f38854b9cae25d17077307d3c1d616ab234/preinstall/installer/ubuntu.sh#L106


Just proxy will created

**Comment by aelfa** [comment]
> 
> 
> https://github.com/dockserver/dockserver/blob/241f2f38854b9cae25d17077307d3c1d616ab234/preinstall/installer/ubuntu.sh#L106
> 
> Just proxy will created

We need to create the `proxy-basic` network for socket-proxy.

**Comment by drag0n141** [comment]
> > https://github.com/dockserver/dockserver/blob/241f2f38854b9cae25d17077307d3c1d616ab234/preinstall/installer/ubuntu.sh#L106
> > 
> > Just proxy will created
> 
> We need to create the `proxy-basic` network for socket-proxy.

Its now added.

**Comment by doob187** [comment]
Each docker-compose  command 
Need to changed to  docker compose

Mass edit  will faster replace this

**Comment by doob187** [comment]
#major

**Pull request merged**